hugo/assets/scss/components/blogs.scss

```diff
@@ -0,0 +1,92 @@
+@import '../config/colors';
+@import '../config/sizes';
+@import '../mixins/container';
+@import '../mixins/list-reset';
+@import '../mixins/stretch';
+
+.blogs {
+    &__list {
+        @include list-reset;
+
+        display: grid;
+        gap: 2rem;
+    }
+
+    @include screen($screen-simple) {
+        &__list {
+            grid-template-columns: 1fr 1fr;
+        }
+
+        &__item {
+            padding: 1rem 0;
+
+            .teaser {
+                .media {
+                    height: 130px;
+                }    
+            }
+
+            &:first-child {    
+                grid-column: 1 / -1;
+
+                .teaser {
```

Comment:
teaser shouldn't be styles by the list itself. You can add a modifiier to teaser which handles the featured teaser styles (add a class in the blog list when index = 1 (or 0 not sure hugo range is 0 indexed) teaser--featured or something? or teaser--highlight